GameData/RP-0/Tree/ECM-Engines.cfg

```diff
@@ -370,10 +370,12 @@
     RL10A-3-3-Lunex = 50000,RL10A-3-3,throttlingTP
     RL10A-3-3A = 30000,RL10A-3-3
     RL10A-4 = 45000,RL10A-3-3A
-    RL10A-4-1-2 = 50000,RL10A-4
+    RL10A-4-1 = 50000,RL10A-4
+    RL10A-4-2 = 50000,RL10A-4-1
+    RL10A-4-2N = 50000,RL10A-4-2
     RL10A-5 = 45000,RL10A-4
-    RL10B-2 = 80000,RL10A-4-1-2
-    RL10C-1 = 60000,RL10A-4-1-2
+    RL10B-2 = 80000,RL10A-4-1
+    RL10C-1 = 60000, RL10A-4-2N
```

Comment:
Couple risks with those.  Note that the upgrade path to RL10C does not go as much through RL10B.  10B derives from A-4-1 here, 10C derives from A-4-2.  The other risk is by destroying the 10A-4-1-2 ECM, any mid-career setup might be very unhappy.